[0m22:17:41.523199 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f4d684cd840>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f4d66c1c910>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f4d66c1c970>]}


============================== 22:17:41.551853 | 1889b090-dbc5-4ca2-969d-6f76b2000185 ==============================
[0m22:17:41.551853 [info ] [MainThread]: Running with dbt=1.8.4
[0m22:17:41.553497 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/farhanriyandi/.dbt', 'version_check': 'True', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'invocation_command': 'dbt init my_project', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m22:17:41.780547 [debug] [MainThread]: Starter project path: /mnt/e/DBT-Alterra4/DBT-DEMO/.venv/lib/python3.10/site-packages/dbt/include/starter_project
[0m22:17:42.056636 [info ] [MainThread]: 
Your new dbt project "my_project" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m22:17:42.058052 [info ] [MainThread]: Setting up your profile.
[0m22:19:04.680705 [info ] [MainThread]: Profile my_project written to /home/farhanriyandi/.dbt/profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m22:19:04.686128 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 83.24822, "process_user_time": 1.810587, "process_kernel_time": 0.796279, "process_mem_max_rss": "89916", "process_in_blocks": "15672", "process_out_blocks": "8"}
[0m22:19:04.688227 [debug] [MainThread]: Command `dbt init` succeeded at 22:19:04.687991 after 83.25 seconds
[0m22:19:04.689655 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f4d684cd840>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f4d68817100>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f4d66c1c220>]}
[0m22:19:04.691031 [debug] [MainThread]: Flushing usage events
[0m07:11:09.979277 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f42c2ab9660>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f42c24f99c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f42c1365fc0>]}


============================== 07:11:10.008113 | ab88b841-1eab-49ef-9f48-e1e5df2736db ==============================
[0m07:11:10.008113 [info ] [MainThread]: Running with dbt=1.8.4
[0m07:11:10.009930 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'version_check': 'True', 'profiles_dir': '/home/farhanriyandi/.dbt', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'static_parser': 'True', 'invocation_command': 'dbt run', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m07:11:10.029964 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /mnt/e/DBT-Alterra4/DBT-DEMO/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m07:11:10.033428 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.13270687, "process_user_time": 1.785493, "process_kernel_time": 0.273964, "process_mem_max_rss": "85348", "process_in_blocks": "39544", "command_success": false, "process_out_blocks": "0"}
[0m07:11:10.035256 [debug] [MainThread]: Command `dbt run` failed at 07:11:10.035083 after 0.13 seconds
[0m07:11:10.036561 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f42c2ab9660>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f42c13662f0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f42c1365fc0>]}
[0m07:11:10.037800 [debug] [MainThread]: Flushing usage events
[0m13:30:03.259467 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8a5d82d7e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8a5db577c0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8a5db57760>]}


============================== 13:30:03.300647 | b86cf81d-e8e0-4022-bd7a-7f68fe42825a ==============================
[0m13:30:03.300647 [info ] [MainThread]: Running with dbt=1.8.4
[0m13:30:03.303250 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/farhanriyandi/.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m13:30:03.305139 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /mnt/e/DBT-Alterra4/DBT-DEMO/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m13:30:03.309725 [debug] [MainThread]: Resource report: {"command_name": "run", "command_wall_clock_time": 0.17123386, "process_user_time": 2.111532, "process_kernel_time": 0.700197, "process_mem_max_rss": "85032", "process_in_blocks": "17768", "command_success": false, "process_out_blocks": "0"}
[0m13:30:03.310972 [debug] [MainThread]: Command `dbt run` failed at 13:30:03.310832 after 0.17 seconds
[0m13:30:03.311984 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8a5d82d7e0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8a5db57850>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7f8a5db57790>]}
[0m13:30:03.313016 [debug] [MainThread]: Flushing usage events
